### PR TITLE
Save, jwt and api modifications

### PR DESCRIPTION
**Pull request by tarunluthra123**
@jatinkatyal13 @BipinKalra

### CODE REVIEW

File: src/components/layout/navbar.tsx [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
+  };
+
+  const history = useHistory();
 
   const logout = () => {
     dispatch(logoutUser());
+    clearJwt();
   };
 
+  async function saveCode() {
```
Comment (jatinkatyal13) [inline src/components/layout/navbar.tsx:30]: can we do task here as well? Just for the sake of having a running state and maybe we can show Saving on the button as well using that.

File: src/initializers/code.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  }
+  const queryParams = new URLSearchParams(location.search);
+  const id = queryParams.get('id');
+
+  if (!id) {
+    return;
+  }
+
+  fetchCode(id);
```
Comment (jatinkatyal13) [inline src/initializers/code.ts:27]: why a function call inside a function? What's the benefit?

File: src/components/layout/navbar.tsx (hunk 1 above)
Comment (tarunluthra123) [reply src/components/layout/navbar.tsx:31]: Fixed.

File: src/initializers/code.ts (hunk 2 above)
Comment (tarunluthra123) [reply src/initializers/code.ts]: Fixed.

**Pull request merged**